deployment/keystone/changeset/operations/contracts/deploy_contracts_seq.go

```diff
@@ -19,6 +21,7 @@ type DeployKeystoneContractsSequenceInput struct {
 	ForwardersSelectors   []uint64
 	DeployVaultOCR3       bool
 	DeployEVMOCR3         bool
+	EVMChainIDs           map[uint64]uint64
```

Comment:
do you think it would make sense to add either a type alias or a comment hat would explain what the key and value represents?